pybossa/forms/dynamic_forms.py

```diff
@@ -31,9 +31,12 @@ def __init__(self, *args, **kwargs):
         pass
 
     if data_access_levels:
+        kw={'checked': not obj or obj.amp_store}
+        print(kw)
```

Comment:
Cleanup. :)

Consider removeing these unused lines?

```python
kw={'checked': not obj or obj.amp_store}
print(kw)
```

Reply:
Oop sorry, that's debugging message. Cleaning up.